client/blocks/product-purchase-features-list/jetpack-site-accelerator.jsx

```diff
@@ -0,0 +1,30 @@
+/** @format */
```

Comment:
This is a Jetpack 4.7 feature. Do we want a version check in order to assure that we're not displaying this for older versions of Jetpack?

Reply:
That would be cool! Could you point me to a resource where I can get this JP version check? Thanks!

Reply:
Sure thing! Here is an example: https://github.com/Automattic/wp-calypso/blob/7b91038e6b42d9fc63722cdc051d76e95e5ba698/client/my-sites/site-settings/manage-connection/data-synchronization.jsx#L49

Note that you'll likely want to do this check in the top level (in ` client/blocks/product-purchase-features-list/index.jsx`) because otherwise if we return an empty box, we might mess up with the calculations inside this page.

Reply:
Yep, gotcha! Thanks again, will push a commit in a few minutes.